climada/util/yearsets.py

```diff
@@ -165,14 +165,8 @@ def sample_from_poisson(n_sampled_years, lam, seed=None):
     """
     if seed is not None:
         np.random.seed(seed)
-    if lam != 1:
-        events_per_year = np.round(np.random.poisson(lam=lam,
-                                                     size=n_sampled_years)).astype('int')
-    else:
-        events_per_year = np.ones(len(n_sampled_years))
-
+    return np.round(np.random.poisson(lam=lam, size=n_sampled_years)).astype('int')
```

Comment:
I am not sure to understand. This changes the behaviour of the method. Is this desired?